applications/plot_simtel_histograms.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/python3
+
+'''
+    Summary
+    -------
+    This application prints sim_telarray histograms into pdf file.
+    It accepts multiple lists of histograms files that are plot together,
+    side-by-side. Each histogram is plotted in a page of the pdf.
+
+
+    Command line arguments
+    ----------------------
+    file_lists (str, required)
+        Text file containing the list of sim_telarray histogram files to be plotted. \
+        Multiple text files can be given.
+    output (str, required)
+        File name for the pdf output (without extension).
+    verbosity (str, optional)
+        Log level to print (default=INFO).
+
+    Example
+    -------
+    .. code-block:: console
+
+        python applications/plot_simtel_histograms.py -l list_test1.txt list_test2.txt -o histograms_comparison
+'''
+
+import logging
+import argparse
+import matplotlib.pyplot as plt
+from matplotlib.backends.backend_pdf import PdfPages
+
+import simtools.util.general as gen
+from simtools.simtel.simtel_histograms import SimtelHistograms
+
+
+if __name__ == '__main__':
+
+    parser = argparse.ArgumentParser(
+        description=(
+            'Plots sim_telarray histograms.'
+        )
+    )
+    parser.add_argument(
+        '-l',
+        '--file_lists',
+        help='File containing the list of histogram files to be plotted.',
+        nargs='+',
+        type=str,
+        required=True
+    )
+    parser.add_argument(
+        '-o',
+        '--output',
+        help='File name for the pdf output (without extension).',
```

Comment:
Would be nice if the application would allow both with and without the `.pdf` extension.

Reply:
Good idea. Done